GTG/gtk/preferences.py

```diff
@@ -17,304 +17,50 @@
 # this program.  If not, see <http://www.gnu.org/licenses/>.
 # -----------------------------------------------------------------------------
 
-""" The Preferences Dialog for configuring GTG """
+""" The Preferences Window for configuring GTG """
 
 import os
-import shutil
 
 from gi.repository import Gtk
-from gi.repository.Gdk import Color
-from xdg.BaseDirectory import xdg_config_home
 
-from GTG import info
-from GTG.core.translations import _
-from GTG.gtk import ViewConfig
-from GTG.gtk import help
-from GTG.tools import shortcut
+from GTG.gtk.general_preferences import GeneralPreferences
+from GTG.core.dirs import UI_DIR
 
-AUTOSTART_DIRECTORY = os.path.join(xdg_config_home, "autostart")
-AUTOSTART_FILE = "gtg.desktop"
-AUTOSTART_PATH = os.path.join(AUTOSTART_DIRECTORY, AUTOSTART_FILE)
 
+class Preferences(object):
+    """ Show preference window """
```

Comment:
Describe that this is just a "framework" that will take care of switching among different preferences UI. It might be worth to mention that it gets those widgets to display via `get_ui` method.